Documentation/DevelopingPackages.md

```diff
@@ -37,13 +38,17 @@ and you need to work on that package simultaneously
 then you have several options:
 
  1. **Edit the sources the package manager clones**
-    
+
```

Comment:
I've watched the two previews and I don't see any differences.
By the way those spaces are not there for the point 2 of the same list